File: c-plus-plus/btfullhouse/buoi5/bai3.cpp
#include<iostream>
using namespace std;
void input(int a[], int n){
	for(int i = 0; i<n; ++i) cin >> a[i];
}
void output(int a[], int n){
	cout << a[0] << " ";
	for(int i = 1; i<n;++i){
		int check = 1;
		for(int j = 0; j<i; ++j){
			if(a[i]<=a[j]){
				check = 0;
				break;
			}
		}
		if(check==1) cout << a[i] << " ";
	}
}
int main(){
	int n;
	cin >> n;
	int a[n];
	input(a,n);
	output(a,n);
}

File: python/buoi2/bai3.py
def check(year):
    if year%4==0: return 1
    elif year%100==0 and year%400==0: return 1
    else: return 0
month = int(input())
year = int(input())
if month<=0 or month >12 or year <0: print("INVALID")
elif month ==1 or month==3 or month==5 or month==7 or month==8 or month==10 or month==12:
    print("31")
elif(month==2):
    if check(year): print("29")
    else: print("28")
else:
    print("30")